src/lib/components/CommunitySelectionModal/CommunityListItem.js

```diff
@@ -20,14 +20,13 @@ export const CommunityListItem = ({ result }) => {
   const linkToCommunityPage = result.links.self_html;
   const linkToLogo = result.links.logo;
   const itemSelected = getChosenCommunity()?.uuid === result.uuid;
-
   return (
     <Item key={result.id} className={itemSelected ? 'selected' : ''}>
-      <Item.Image
-        as={Image}
+      <Image
         size="tiny"
         src={linkToLogo}
         fallbackSrc="/static/images/square-placeholder.png"
+        asItem={true}
```

Comment:
did you check if it works with `as={Item.Image}`?

Reply:
It works, https://github.com/inveniosoftware/react-invenio-forms/pull/132 updated!